common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -117,6 +117,7 @@ def B(*args, **kwargs):
 AUTH_ENTRY_DASHBOARD = 'dashboard'
 AUTH_ENTRY_LOGIN = 'login'
 AUTH_ENTRY_REGISTER = 'register'
+AUTH_ENTRY_ACCOUNT_SETTINGS = 'account_settings'
```

Comment:
There's also code and tests around redirecting to the dashboard once third party auth is complete-- _SOCIAL_AUTH_LOGIN_REDIRECT_URL. I'm not clear if that is still correct (I don't see that I am sent to the dashboard after I link from the account_settings page, so that is good).